stepper.md

```diff
@@ -1,64 +1,214 @@
+8  num speed LSB                           (steps per second)
+9  num speed MSB
+10 [optional] accel LSB                    (acceleration in steps/sec^2)
 11 [optional] accel MSB
-12 [optional] decel LSB (deceleration in 0.01*rad/sec^2)
+12 [optional] decel LSB                    (deceleration in steps/sec^2)
 13 [optional] decel MSB
-14 END_SYSEX            (0xF7)
+14 END_SYSEX                               (0xF7)
+```
+
+**Stepper to**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  to command                              (0x03)
+3  device number                           (0-9)
+4  num target byte1 LSB
+5  num target byte2 MSB
+6  speed LSB                               (steps per second)
+7  speed MSB
+8  [optional] accel LSB                    (acceleration in steps/sec^2)
+9  [optional] accel MSB
+10 [optional] decel LSB                    (deceleration in steps/sec^2)
+11 [optional] decel MSB
+12 END_SYSEX                               (0xF7)
+```
+
+**Stepper enable**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  enable command                          (0x04)
+3  device number                           (0-9)
+4  END_SYSEX                               (0xF7)
+```
+
+**Stepper disable**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  disable command                         (0x05)
+3  device number                           (0-9)
+4  END_SYSEX                               (0xF7)
+```
+
+**Stepper stop**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  stop command                            (0x06)
+3  device number                           (0-9)
+14 END_SYSEX                               (0xF7)
+```
+
+**Stepper reply (sent when a move completes or stop is called)**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  stop reply command                      (0x07)
+3  device number                           (0-9)
+4  position byte 1 LSB                            
+5  position byte 2 MSB                            
+6  END_SYSEX                               (0xF7)
+```
+
+**Stepper limit**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  stop limit command                      (0x08)
+3  device number                           (0-9)
+4  lower limit pin number                  (0-127)
+5  lower limit pin number                  (0-127)
+6  END_SYSEX                               (0xF7)
+```
+
+**MultiStepper configuration**
+```
+0  START_SYSEX                              (0xF0)
+1  Stepper Command                          (0x62)
+2  config command                           (0x20)
+3  group number                             (0-4) (supports up to 5 groups)
```

Comment:
I couldn't find anything in the documentation or the forums, but I see there is nothing in the code to guard against it. I had chosen to limit it to 5 groups because in my mind with 10 steppers you could only create 5 groups, but now I see that you could create any number of combinations so I'll fix that.

BTW, I've had use cases where it made sense to include servos in multiple groups so I think it's a reasonable thing to allow for steppers.